cumulusci/tasks/release_notes/tests/test_parser.py

```diff
@@ -562,12 +563,77 @@ def _create_expected_render(self, issue_number, issue_title, link_pr):
 
 class TestInstallLinkParser:
     def test_no_package_version(self):
-        generator = mock.Mock(link_pr=True)
-        generator.version_id = None
+        generator = mock.Mock(link_pr=True, version_id=None)
         parser = InstallLinkParser(generator, "Title")
         parser.parse("abc")
         assert parser.render() == ""
 
+    def test_package_release_info_no_version_id(self):
+        generator = mock.Mock(link_pr=True, version_id=None, release_info=True)
+        parser = InstallLinkParser(generator, "Title")
+        parser.parse("abc")
+        assert "# Title" in parser.render()
+        assert "# Installation Info" in parser.render()
+        assert "## Push Schedule" in parser.render()
+        assert f"Sandbox orgs: {generator.sandbox_date}" in parser.render()
+        assert f"Production orgs: {generator.production_date}" in parser.render()
+        assert "## Installation URL" not in parser.render()
```

Comment:
I think it would be better to assert that the entire rendered text matches what we expect (as a multi-line string). Doing it this way doesn't assert that the lines are in the correct order.